### ISSUE TITLE
*breaking* changes - next version 0.8.1 or jump to v0.9.0

### ISSUE THREAD

**Issue by danielpeintner** [eclipse/thingweb.node-wot#738]
Note: since we are introducing some *breaking* changes in https://github.com/eclipse/thingweb.node-wot/pull/731 @relu91 is suggesting to jump to v0.9.0
see https://github.com/eclipse/thingweb.node-wot/pull/731#discussion_r844984781

EDIT: One downside is a bit that we loose the connection between  "wot-typescript-definitions" versioning

_Originally posted by @danielpeintner in https://github.com/eclipse/thingweb.node-wot/issues/727#issuecomment-1092634350_

**Comment by danielpeintner** [comment]
My take is that since we only remove deprecated methods I personally would go for version 0.8.1.

What do others think?

**Comment by egekorkan** [comment]
From my point of view, this is rather a small update (just removing the consumed thing methods right?) thus more inclined for 0.8.1 but this would, of course, contradict with semver. There have been previous cases of semver getting broken, like removing writeproperty from exposed thing but then bringing it back (couldn't find the issue). One thing that would actually bother me would be that outsiders getting confused since the perceived change from 7->8 and 8->9 is the same when seen from the version number but the implementation change is not, especially now since 8 is just published. For the migration guides, we would then need to directly say 0.9 and also deprecate 0.8 from npm. Also, all the marketing posts we have done would look outdated now :/

To summarize:
- I am sort of ok to go for 0.9 if we deprecate 0.8 from npm
- I am ok to go for 0.8.1 while putting a note that removing the inherited functions was a breaking change but we did it quickly after 0.8.0 and would affect only a small number of people

**Comment by danielpeintner** [comment]
closing since
* [v0.8.1](https://github.com/eclipse/thingweb.node-wot/releases/tag/v0.8.1) has been released
* see also https://github.com/eclipse/thingweb.node-wot/issues/770
